index.js

```diff
@@ -379,3 +379,112 @@ exports.requestMultiItemBasic = (itemIds, cb) => {
       })
   }
 }
+
+
+// Just like auth except in case of a successful call, returns a promise that resolves to the value of the
+// callback
+
+exports.promiseAuth = (cb) => {
+  if (!exports.credsKey || !exports.credsSecret || !exports.credsBase) {
```

Comment:
This is not a big deal for the PR but it seems we don't actually need to have any of these variables be part of `exports`? Those secrets should be kept within the file and not exported out.

Reply:
Not sure if I am understanding correctly but I think this check is basically to see if loadconfig has been called